tests/node_functions.py

```diff
@@ -88,6 +88,23 @@ def test_node_pow(self):
         x = F.input(input_arr)
         self.assertTrue(((x ** 0x7fffffff).to_ndarrays()[0] == np.array([1, -1])).all())
         self.assertTrue(((x ** -0x80000000).to_ndarrays()[0] == np.array([1, 1])).all())
-        self.assertTrue(np.isnan((x ** 0x80000000).to_ndarrays()[0]).any())
-        self.assertTrue(np.isnan((x ** -0x80000001).to_ndarrays()[0]).any())
+        # NOTE(vbkaisetsu):
```

Comment:
These tests should not be supported because 0x80000000 and -0x80000001 are out-of-range of int32_t and results of the core library is basically undefined.